intezer_analyze_cli/commands.py

```diff
@@ -340,6 +343,78 @@ def upload_multiple_offline_endpoint_scans(offline_scans_root_directory: str,
         click.echo(f'{failed_number} offline endpoint scans failed to send')
 
 
+def send_phishing_emails_from_directory_command(path: str,
+                                                ignore_directory_count_limit: bool = False):
+    success_number = 0
+    failed_number = 0
+    unsupported_number = 0
+    emails_dates = []
+
+    for root, dirs, files in os.walk(path):
+        files = [f for f in files if not is_hidden(os.path.join(root, f))]
+        dirs[:] = [d for d in dirs if not is_hidden(os.path.join(root, d))]
+
+        number_of_files = len(files)
+        if not ignore_directory_count_limit:
+            utilities.check_should_continue_for_large_dir(number_of_files, default_config.unusual_amount_in_dir)
```

Comment:
shouldn't the limit need to be over all the files and not for this itteration?

Reply:
@almogch almog I thinks that in every iteration we can pass te limit as it can have subdirctories